Lib/idlelib/editor.py

```diff
@@ -188,6 +189,9 @@ def __init__(self, flist=None, filename=None, key=None, root=None):
         vbar['command'] = text.yview
         vbar.pack(side=RIGHT, fill=Y)
         text['yscrollcommand'] = vbar.set
+        hbar['command'] = text.xview
+        hbar.pack(side='bottom', fill='x')
```

Comment:
I think the `fill='x'` should change to `fill=X`.

Reply:
The more important change is that vbar is now gridded (hence conflict) and hbar must be also.  textview.ScrollableTextFrame (wrap=None) uses `sticky=EW`.